src/complex-list-field/complex-list-field.component.ts

```diff
@@ -100,10 +100,19 @@ export class ComplexListFieldComponent extends AbstractListFieldComponent implem
     }
   }
 
-  onFieldAdd(field: string, index: number) {
+  onFieldAdd(index: number, field: string) {
     this.keys[index].push(field);
   }
 
+  deleteField(index: number, field: string) {
+    // remove it the field from the element of list (record)
```

Comment:
This comment is a bit difficult to understand as is, might be better without it.